### ISSUE TITLE
WCAG Issue on main slider bottom pills / bottom indicator

### ISSUE THREAD

**Issue by iqbalhood** [topcoder-archive/beta-testing-dashboard#378]
Element bottom pills/bottom indicator on the main slider is too small, and the color contrast with some banner does not meet the Accessibility standard (WCAG), Need to be replaced with bigger bottom pills or add an overlay layer to increase contrast in various conditions the banner image will be displayed
<img width="592" alt="Screen Shot 2021-09-16 at 00 49 49" src="https://user-images.githubusercontent.com/1563756/133484478-6e2f5231-c0ce-4673-9da9-c6049e528074.png">

**Comment by kkartunov** [comment]
Not true bug in sense of broken functionality.
